src/components/rating/rating.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import injectSheet from 'react-jss';
+import cn from 'classnames';
+import { Icon } from '../../';
+import styles from './rating.styles';
+
+/* eslint-disable react/no-array-index-key */
+const Rating = ({ icon: RenderIcon = Icon.Star, size, rating, maxRating, classes }) => (
+  <div className={classes.rating}>
+    {[...Array(maxRating)].map((_, index) => {
+      const classNames = cn({
+        [classes.active]: index < rating,
+        [classes.inactive]: index >= rating,
+        [classes.ratingIcon]: true,
+      });
+      return (
+        <RenderIcon
```

Comment:
Rename to `Icon` maybe? `RenderIcon` sounds a bit weird.

Reply:
Collides with the imported Icon